lib/Net/SSLeay.pod

```diff
@@ -9933,6 +9933,16 @@ Or alternatively you can just use the following convenience functions:
 
 =head1 KNOWN BUGS AND CAVEATS
 
+LibreSSL versions in the 3.1 - 3.3 series contain a TLS 1.3 implementation that
+is not fully compatible with the libssl API, but is still advertised during
+protocol auto-negotiation. If you encounter problems or unexpected behaviour
+with SSL or SSL_CTX objects whose protocol version was automatically negotiated
+and libssl is being provided by any of these versions of LibreSSL, consider
+setting a maximum protocol version (with C<Net::SSLeay::set_max_proto_version()>
+or C<Net::SSLeay::CTX_set_max_proto_version()>) before establishing a
+connection. The first stable LibreSSL version with a fully libssl-compatible TLS
+1.3 implementation is 3.4.1.
+
```

Comment:
Would it be stating the obvious to say that if there are problems with automatic protocol, then it could mean that _TLS 1.3 was attempted_? In other words, suggest using `*set_max_proto_version()` to TLS 1.2 or lower.

It could easily be something else too, so on the other hand I'd avoid trying to sound too specific that this is the only possible problem.